R/reliabilityAnalysisFrequentist.R

```diff
@@ -0,0 +1,606 @@
+
+  if (!is.null(jaspResults[["itemTableF"]]) || !any(model[["derivedOptions"]][["itemDroppedSelectedF"]])) {
+    print("jaspResults[['hasItemTableF']] from state or not wanted")
+    return()
+  }
+  
+  derivedOptions <- model[["derivedOptions"]]
+  # fixes issue that unchecking the scale coefficient box, does not uncheck the item-dropped coefficient box:
+  for (i in 1:5) { 
+    if (!derivedOptions[["selectedEstimatorsF"]][i]) {
+      derivedOptions[["itemDroppedSelectedF"]][i] <- derivedOptions[["selectedEstimatorsF"]][i]
+    }
+  }
+  itemDroppedSelectedF <- derivedOptions[["itemDroppedSelectedF"]]
+  # order <- derivedOptions[["order_item"]]
+  estimators <- derivedOptions[["namesEstimators"]][["tables_item"]]
+  overTitle <- "If item dropped"
+  
+  itemTableF <- createJaspTable("Frequentist Individual Item Reliability Statistics")
+  itemTableF$dependOn(options = c("variables",
+                                  "mcDonaldScalef", "alphaScalef", "guttman2Scalef", "guttman6Scalef", "glbScalef", 
+                                  "averageInterItemCor", "meanScale", "sdScale",
+                                  "mcDonaldItemf",  "alphaItemf",  "guttman2Itemf", "guttman6Itemf", "glbItemf",
+                                  "reverseScaledItems", "meanItem", "sdItem", "itemRestCor", "missingValuesf", 
+                                  "omegaEst", "alphaMethod", "setSeed", "seedValue"))
+  itemTableF$addColumnInfo(name = "variable", title = "Item", type = "string")
+  
+  idxSelectedF <- which(itemDroppedSelectedF)
+  coefficients <- derivedOptions[["namesEstimators"]][["coefficients"]]
+  for (i in idxSelectedF) {
+    if (estimators[i] %in% coefficients) {
+      itemTableF$addColumnInfo(name = paste0("pointEst", i), title = estimators[i], type = "number", 
+                               overtitle = overTitle)
+    } else {
+      itemTableF$addColumnInfo(name = paste0("pointEst", i), title = estimators[i], type = "number")
+    }
+  }
+  
+  relyFit <- model[["relyFit"]]
+  if (!is.null(relyFit)) {
+    tb <- data.frame(variable = model[["itemsDropped"]])
+    for (i in idxSelectedF) {
+      newtb <- cbind(pointEst = relyFit$freq$ifitem[[i]])
+      colnames(newtb) <- paste0(colnames(newtb), i)
+      tb <- cbind(tb, newtb)
+      
+    }
+    itemTableF$setData(tb)
+    
+  } else if (length(model[["itemsDropped"]]) > 0) {
+    itemTableF[["variables"]] <- model[["itemsDropped"]]
+  }
+  
+  jaspResults[["itemTableF"]] <- itemTableF
+  return()
+}
+
+# once the package is updated check this again and apply:
+.freqentistReliabilitySingleFactorFitTable <- function(jaspResults, model, options) {
+
+  if (!options[["fitMeasures"]] || !options[["mcDonaldScalef"]])
+    return()
+  if (!is.null(jaspResults[["fitTable"]]) || !options[["fitMeasures"]]) {
+    print("jaspResults[['fitTable']] from state or not wanted")
+    return()
+  }
+
+  fitTable <- createJaspTable(sprintf("Fit Measures of Single Factor Model Fit"))
```

Comment:
```suggestion
  fitTable <- createJaspTable(gettext("Fit Measures of Single Factor Model Fit"))
```
no need for the f in sprintf right?